docs/configuration/monitoring.md

```diff
@@ -1,77 +1,61 @@
+-   **[Crow](#setting-up-monitoring-using-crow):** A simple, lightweight, and built-in monitoring solution.
+-   **[Prometheus + Grafana + Loki](#setting-up-monitoring-using-grafana-prometheus-and-loki):** A comprehensive third-party monitoring solution, recommended for more advanced usage.
 
-## Setting up monitoring using Crow
+The sections below provide details on how to set up each monitoring solution.
 
-Crow displays information about the projects pending analysis and the jobs currently running on Codacy.
+## Setting up monitoring using Crow
 
-The Crow tool is installed alongside Codacy after the helm chart is deployed to the cluster.
-It can be accessed through the `/monitoring` path of the url pointing to your Codacy
-installation, e.g. `http://codacy.company.org/monitoring`. You must set the
-`global.codacy.crow.url` value in your `values.yaml` file so that anchor links to your
-projects can be properly established inside `crow`.
+Crow displays information about the projects that are pending analysis and the jobs currently running on Codacy.
 
-For example:
+Crow is installed alongside Codacy when the Helm chart is deployed to the cluster and will be available on the `/monitoring` path of your Codacy installation URL, such as `http://<codacy hostname>/monitoring`.
 
-```yaml
-global:
-  codacy:
-    crow:
-      url: "http://codacy.example.com/monitoring"
-```
+Follow the steps below to set up Crow:
 
-### Configuring your credentials
+1.  Download the template file [`values-crow.yaml`](https://github.com/codacy/chart/blob/master/codacy/values-crow.yaml){: target="_blank"}. You can download the template file by running:
 
-We highly recommend that you define a custom password for the Crow installation. You can do this either through the `values.yaml` file or through a `--set` parameter during the Helm installation process. This parameter can be configured as follows:
+    ```bash
+    wget https://raw.githubusercontent.com/codacy/chart/master/codacy/values-crow.yaml
+    ```
 
-* Through a `--set` parameter:
+2.  Use a text editor of your choice to edit the value placeholders as described in the comments:
 
-```yaml
-helm upgrade (...options used to install codacy...) --set crow.config.passwordAuth.password=<--- crow password --->
-```
+    -   Set the correct hostname of your Codacy instance so that Crow correctly generates anchor links to your projects
+    -   Set the username, password, and hostname to access the Crow PostgreSQL database [already created here](../requirements.md#preparing-postgresql-for-codacy)
+    -   Set the password for accessing Crow
 
-* Through the `values.yaml` file:
+3.  Apply this configuration by performing a Helm upgrade. To do so append `--values values-crow.yaml` to the command [used to install Codacy](../index.md#2-installing-codacy):
```

Comment:
@h314to What do you think of this (hopefully more straightforward) way of configuring Crow?

Reply:
Since we are enabling this by default and it should just be configured in the `values-production.yaml`, should we have this here?

Reply:
No, and now I understand why at first look the Crow configuration seemed to be completely wrong! :sweat_smile: